styles.css

```diff
@@ -0,0 +1,476 @@
+  overflow: hidden;
+}
+
+.topPageDiv {
+  display: none;
+}
+
+.topPage__img {
+  padding: 7px;
+}
+
+.topPage__text {
+  padding: 1px 5px;
+}
+
+.navbar {
+  display: none;
+}
+
+.hamburger {
+  padding: 10px;
+}
+
+.line {
+  width: 25px;
+  height: 5px;
+  background-color: var(--grey);
+  border-radius: 7px;
+  margin: 5px;
+}
+
+.logo {
+  width: 40px;
+}
+
+.nav-link {
+  color: var(--grey);
+}
+
+header {
+  background-image: url(./elements/bckgrnd8.jpeg);
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+  height: 100%;
+}
+
+.main-text {
+  padding: 15% 2% 15% 1%;
+}
+
+.main-text__title {
+  background-image: url(./elements/bckgrndalpha.svg);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  font-size: 46px;
+  text-transform: uppercase;
+  font-weight: 700;
+  -webkit-font-smoothing: antialiased;
+}
+
+.main-text__bio {
+  color: var(--grey);
+}
+
+.main-text__dates,
+.main-text__location {
+  color: var(--grey);
+  font-weight: bolder;
+}
+
+.main-text__location {
+  font-weight: 300;
+  font-size: 30px;
+}
+
+#about {
+  background-image: url(./elements/bckgrnd8.jpeg);
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+}
+
+.about-text {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  padding: 90px 10px;
+}
+
+.about-text__title {
+  display: flex;
+  background-image: url(./elements/bckgrnd6.jpeg);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  font-size: 46px;
+  text-transform: uppercase;
+  font-weight: 700;
+}
+
+.text-separator {
+  width: 50px;
+  height: 4px;
+  background: var(--orange);
+  border-radius: 7px;
+  margin: 10px;
+}
+
+.about-text__bio {
+  padding: 20px;
+  border: 1px solid var(--grey);
+  border-radius: 3px;
+  background: rgba(211, 211, 211, 0.666);
+  color: var(--black);
+}
+
+.about-text__footer {
+  color: var(--grey);
+}
+
+.logo-section {
+  border-bottom: 2px solid var(--grey);
+}
+
+.about-logo {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 8px;
+  padding: 50px 5px;
+  text-align: center;
+}
+
+.about-logo__images {
+  width: 95%;
+  display: flex;
+  padding: 20px;
+  flex-direction: row;
+  border: 1px solid darkgrey;
+}
+
+.about-logo__images img {
+  width: 100px;
+  height: 100px;
+}
+
+.past-summits-section {
+  border-bottom: 2px solid var(--grey);
+}
+
+.summit-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 5px;
+  padding: 30px;
+  text-align: center;
+}
+
+.summit-section__images {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 20px;
+}
+
+.summit-section img {
+  width: 97%;
+}
+
+#program {
+  background-image: url(./elements/prgrmssctn1.jpeg);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100%;
+  padding: 5%;
+}
```

Comment:
- [ ] As requested before it is required to use the same background as the original design :+1: . [Here](https://ingdavidtm.github.io/capstone-biotechnology/images/patternBackground.png) you can find the required background.
- [ ] Also, you should use a transparent background for each card :+1: 

**Your design**

![Screenshot from 2023-02-09 06-38-31](https://user-images.githubusercontent.com/105117832/217802736-a7505dd2-8b51-4d20-a993-e3b8e4837096.png)

**Template**

![Screenshot from 2023-02-09 06-38-13](https://user-images.githubusercontent.com/105117832/217802786-3e8ab249-bc3d-4370-979f-04ccef5cc19d.png)